src/transport/url.js

```diff
@@ -35,9 +35,22 @@ const send = ({ uriHandler, messageToURI = defaultMessageToURI } = {}) => {
  *  @return   {Object}   A response object if repsonse is available, otherwise null.
  */
 const getResponse = () => {
-  const res = window.location.hash.slice(1)
-  window.location.hash = ''
-  return parseResponse(res)
+  const hash = window.location.hash.startsWith('?') ? window.location.hash.slice(1) : window.location.hash
```

Comment:
Do we send a `?` across from the mobile app? We shouldn't.

Reply:
The hash will always start with a `#`

Reply:
No, we do not send a `?` from the mobile app.  I put this line in there because the existing tests all mock `global.window.location.hash` to start with `?`.  You are right though, I will fix this line and change the existing hash strings in the test.